src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Add.PackageReference
+{
+    internal class LocalizableStrings
```

Comment:
Ok, given @srivatsn response, we will take it and I will update the xlf files once this is in.